frontend/packages/helm-plugin/src/components/forms/ProjectHelmChartRepository/projecthelmchartrepository-validation-utils.ts

```diff
@@ -0,0 +1,29 @@
+import { TFunction } from 'i18next';
+import * as yup from 'yup';
+import { nameRegex } from '@console/shared/src';
+
+const urlRegex = /^((https?:?):\/\/:?)/;
```

Comment:
Don't think we need the capture groups if we aren't using them.
And can also check for text after https
https://regex101.com/r/g2k6tk/1
```suggestion
const urlRegex = /^https?:?:\/\/:?.+/;
```